### ISSUE TITLE
Getting this warning: [When linting something other than CSS, you should install an appropriate syntax, e.g. "@stylelint/postcss-css-in-js", and use the "customSyntax" option]

### ISSUE THREAD

**Issue by TowhidKashem** [kristerkari/stylelint-config-react-native-styled-components#37]
Hey @kristerkari, thanks a lot for both these react native plugins!

one question/concern I had was when running stylelint I get the following warning:

`When linting something other than CSS, you should install an appropriate syntax, e.g. "@stylelint/postcss-css-in-js", and use the "customSyntax" option`

Is this something I can safely ignore or should I be installing some additional packages and using the `customSyntax` option in the stylelint config?

I get this just running the default stylelint command on my project's tsx files.

**Comment by kristerkari** [comment]
@TowhidKashem you should not ignore it, so if you just follow those instructions in your config, everything should work fine :)

**Comment by TowhidKashem** [comment]
I installed `@stylelint/postcss-css-in-js` and added it to the config file like so:

```
module.exports = {
  customSyntax: '@stylelint/postcss-css-in-js',
  processors: ['stylelint-processor-styled-components'],
  extends: ['stylelint-config-react-native-styled-components']
};
```

But get this error:

`Error: Cannot resolve custom syntax module "@stylelint/postcss-css-in-js". Check that module "@stylelint/postcss-css-in-js" is available and spelled correctly.`

`Caused by: Error: Cannot find module 'postcss-syntax/load-syntax'`

Looks like it's a issue with the lib:

https://github.com/stylelint/postcss-css-in-js/issues/76

For now I was able to supress the warnings by using `customSyntax: 'postcss'`. Doesn't seem to be resulting in any odd behavior for now 🤞
